rest/taskana-rest-spring/src/main/java/pro/taskana/rest/common/TaskanaRestExceptionHandler.java

```diff
@@ -37,7 +37,8 @@
 @ControllerAdvice
 public class TaskanaRestExceptionHandler extends ResponseEntityExceptionHandler {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(TaskanaRestExceptionHandler.class);
+  private static final Logger REST_EXCEPTION_HANDLER_LOGGER = LoggerFactory.getLogger(
```

Comment:
What's wrong with the name LOGGER?

Reply:
This is a suggestion by sonarcloud to prevent a mixup with the field of the superclass.